tests/server/server-executeQuery-004.phpt

```diff
@@ -7,11 +7,14 @@ MongoDB\Driver\Manager::executeDelete() multiple documents
 require_once __DIR__ . "/../utils/basic.inc";
 
 $manager = new MongoDB\Driver\Manager(STANDALONE);
+$server = $manager->executeQuery(NS, new MongoDB\Driver\Query(array()))->getServer();
```

Comment:
The test is named `server-executeQuery-004` and yet its title was "Manager::executeDelete()" and it was only ever calling `Manager::executeInsert()` and `Manager::executeQuery()`. This all seemed very incorrect, so I fixed things to actually invoke the server methods.

`$manager->executeQuery(NS, new MongoDB\Driver\Query(array()))->getServer();` was something I borrowed from some other server tests. I suppose I could just have used `Manager::selectServer()` there to obtain the primary.